plugins/Morpheus/stylesheets/base/icons.css

```diff
@@ -403,3 +403,9 @@
 .icon-new_releases:before {
   content: "\e902";
 }
+.icon-arrow-left2:before {
```

Comment:
for consistency we should maybe name the icons with a `-`.
e.g. `icon-arrow-left-2`, like you used it in demo.twig, which currently doesn't work as they have another name..